components/server/src/workspace/workspace-starter.ts

```diff
@@ -880,6 +880,16 @@ export class WorkspaceStarter {
                 { workspace: workspace.id, instance: instance.id },
                 `preStartWorkspaceModifyHook: Got an error response from extensionService: ${response.getError()}`,
             );
+            // ! check if error === IMAGE_ARCH_MISMATCH_ERROR, if yes we should not throw an error
+            const IMAGE_ARCH_MISMATCH_ERROR = "DS: Image arch mismatch";
```

Comment:
is there a better place to define the constant ?

Reply:
Gitpod's not really using a constants file. The way I like to handle is to define the variables close to where they are being used. Right now we only need the IMAGE_ARCH_MISMATCH_ERROR in one place. If we are to use it in multiple places in this file, we can move it to top of file then. 

Don't you agree? 
Defining constants right at the top/start should only be the case when you need them in global scope, no?

But yeah the better solution would be to define a new constants file altogether, so I have done that anyways.